### PR TITLE
Some readme edits

### PR DESCRIPTION
**Pull request by nikhilro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,15 +10,17 @@ Currently, this repo contains implementations of the rerankers for [CovidQA](htt
 
 ## Installation
 
-1. For pip, do `pip install pygaggle`. If you prefer Anaconda, use `conda env create -f environment.yml && conda activate pygaggle`.
+0. Clone this repo `git clone git@github.com:castorini/pygaggle.git`
```
Comment (rodrigonogueira4) [inline README.md:13]: It seems confusing to me that we need to clone AND do a pip install.. maybe move the clone instruction to the eval section, so it is needed only by users who want to run the evaluation scripts?

Comment (lintool) [reply README.md:13]: FWIW, we should distinguish between (1) just using pygaggle, in which case `pip` is sufficient, and (2) developing/evaluating, in which we need to clone the entire repo etc,.

Comment (nikhilro) [reply README.md:13]: Yea you are both right. Will change and clarify.

File: README.md [hunk 2]
```diff
@@ -8,17 +8,28 @@ It was designed for tight integration with [Pyserini](http://pyserini.io/), but
 
 Currently, this repo contains implementations of the rerankers for [CovidQA](https://github.com/castorini/pygaggle/blob/master/data/) on CORD-19, as described in ["Rapidly Bootstrapping a Question Answering Dataset for COVID-19"](https://arxiv.org/abs/2004.11339).
 
+## Usage
+
+0. Install via PyPI `pip install pygaggle`. Requires [Python 3.6+](https://www.python.org/downloads/)
```
Comment (nikhilro) [inline README.md:13]: I don't have context on what a simple usage might be. Hoping we can add something similar to the example in https://github.com/castorini/pyserini

**Comment by nikhilro** [comment]
> (1) just using pygaggle, in which case pip is sufficient, and (2) developing/evaluating, in which we need to clone the entire repo

@rodrigonogueira4 @lintool do you guys think this helps distinguish between the two?

**Comment by rodrigonogueira4** [comment]
> > (1) just using pygaggle, in which case pip is sufficient, and (2) developing/evaluating, in which we need to clone the entire repo
> 
> @rodrigonogueira4 @lintool do you guys think this helps distinguish between the two?

I think the installation instructions should be kept the same as before, i.e., pip install pygaggle, install  torch, Download the index, etc...
Then, inside the section "## Running rerankers on CovidQA", we add the following instruction: "Before running these evaluations, you need to clone this repo `git clone ...`".

**Comment by nikhilro** [comment]
Ah okay, I guess those are needed even if you don't intend to run the evaluations. Sorry I have no clue about this...Will change

**Pull request merged**